Modules/Brand/Http/Services/BrandService.php

```diff
@@ -533,9 +603,33 @@ private function imageUpload(int $brand, $image, $previousFile, $replaceable): S
     public function updateAccount(array $requestData): array
     {
 
-        $user = User::find($requestData['user_id']);
+        $userId = auth()->user()->id;
+        $user = User::find($userId);
         $user->first_name = $requestData['first_name'];
         $user->last_name = $requestData['last_name'];
+        
+        $user->save();
+
+        $brand = Brand::where('user_id', $userId)->first();
```

Comment:
Make relationship of user and brand and get the relational data with user query.